unit_tests/test_exception_handler.py

```diff
@@ -47,30 +47,9 @@ def test_given_exception_with_display_message_when_assemble_uncaught_exception_t
 def test_uncaught_exception_handler():
     cmd = "from airbyte_cdk.logger import init_logger; from airbyte_cdk.exception_handler import init_uncaught_exception_handler; logger = init_logger('airbyte'); init_uncaught_exception_handler(logger); raise 1"
     exception_message = "exceptions must derive from BaseException"
-    exception_trace = (
-        "Traceback (most recent call last):\n"
-        '  File "<string>", line 1, in <module>\n'
-        "TypeError: exceptions must derive from BaseException"
-    )
 
-    expected_log_message = AirbyteMessage(
-        type=MessageType.LOG,
-        log=AirbyteLogMessage(level=Level.FATAL, message=f"{exception_message}\n{exception_trace}"),
-    )
-
-    expected_trace_message = AirbyteMessage(
-        type=MessageType.TRACE,
-        trace=AirbyteTraceMessage(
-            type=TraceType.ERROR,
-            emitted_at=0.0,
-            error=AirbyteErrorTraceMessage(
-                failure_type=FailureType.system_error,
-                message="Something went wrong in the connector. See the logs for more details.",
-                internal_message=exception_message,
-                stack_trace=f"{exception_trace}\n",
-            ),
-        ),
-    )
+    traceback_start = "Traceback (most recent call last):"
```

Comment:
Dropping a note here to confirm that the change to this test looks correct to me - we still check for the presence of certain text in the exception message, but we relax the structure since newer Python versions report in slightly different formats.